benchmarks/ios/visual-effects-ny/shared/src/commonMain/kotlin/HappyNY.kt

```diff
@@ -240,10 +256,14 @@ fun NYContent() {
 
         LaunchedEffect(Unit) {
             while (true) {
-                withFrameNanos {
-                    prevTime = time
-                    time = it
-                }
+                prevTime = time
+                time += 10000000
+                delay(10)
```

Comment:
Not sure how this affects the results, but it is better to use `withFrameNanos` instead of this.

`withFrameNanos` is consistenly called one time each frame. `delay` on the other hand is unpredictable. The actual delay can be 7-13 milliseconds.

Reply:
fixed